tools/CMakeLists.txt

```diff
@@ -336,6 +336,18 @@ foreach(BINARY IN LISTS TOOL_BINARIES)
     endif()
     set_target_properties(${BINARY} PROPERTIES LINK_FLAGS "${JXL_WASM_TOOLS_LINK_FLAGS}")
   endif()
+  
+  # Attach manifest that tells Windows to use UTF-8 for eg. fopen
+  if(WIN32)
+    # Accommodate cl, clang-cl and clang with GNU-like command
+    if(NOT MINGW)
+	  target_sources(${BINARY} PRIVATE utf8.manifest)
+	# Since CMake in MINGW doesn't support linking
+	# .manifest file, do it with .rc file
+	elseif(MINGW)
+	  target_sources(${BINARY} PRIVATE utf8.rc)
+	endif()
```

Comment:
Indentation appears to be messed up here.

Reply:
Yeah, let me see if I can fix it.

Reply:
It still seems to be wrong. I think your editor is turning spaces into tabs.

Reply:
Should be fixed now, had to tweak my VSCode settings.